### PR TITLE
added test for the profile routes

### PR DESCRIPTION
**Pull request by Rupeshiya**
**Added tests for profile routes -> Passing tests**
Screenshot from 2019-10-16 22-49-03

@vaibhavdaren plz review. Used assertion as well.

### CODE REVIEW

File: server/tests/profile.test.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+const express = require('express');
+const app = express();
+const path = require('path');
+const cookie = require('cookie-session');
+const chaiHttp = require('chai-http');
+const server = require('../server');
+const should = chai.should();
+const assert = require('assert');
+var userId = '';
+const otherUserId = '5da73bf3106f2145f348aee2'; // this is the id of other user, so make sure to change it when testing with other userId
```
[the remark this answers is not in the document]
Comment (Rupeshiya) [reply server/tests/profile.test.js]: This should not be in the credential.js, it's a mock dat. Plz think once

File: server/tests/profile.test.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+const assert = require('assert');
+var userId = '';
+const otherUserId = '5da73bf3106f2145f348aee2'; // this is the id of other user, so make sure to change it when testing with other userId 
+var loggedInToken = '';
+chai.use(chaiHttp);
+
+function loginUser(done) {
+  const loginInfo = {
+    email: 'test3@gmail.com',
+    pass: 'password'
```
[the remark this answers is not in the document]
Comment (Rupeshiya) [reply server/tests/profile.test.js]: same here

File: server/tests/profile.test.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+}
+
+//  function to getDetails 
+function getDetails(done){
+  chai.request(server)
+    .post('/profile/getDetails')
+    .set('Authorization', loggedInToken)
+    .send({
+      email: 'test3@gmail.com'
+    })
```
[the remark this answers is not in the document]
Comment (Rupeshiya) [reply server/tests/profile.test.js]: same here

File: server/tests/profile.test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Rupeshiya) [reply server/tests/profile.test.js]: @vaibhavdaren plz give your view

**Comment by ayushnagar123** [comment]
I agree with @Rupeshiya that there is no other way than hardcoding. I guess what we can do is to make test for credential.js as git ignores it while making a pull request. There are chances of buggy pull request get merged

**Comment by vaibhavdaren** [comment]
There are ways but it looks good to me. For now.

**Pull request merged**